src/NodeDocumentationMarkdownGenerator/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
+    "NodeDocumentationMarkdownGenerator": {
+      "commandName": "Project",
+      "commandLineArgs": "fromdirectory -i \"C:\\Users\\SylvesterKnudsen\\Documents\\GitHub\\DynamoRevit\\bin\\AnyCPU\\Release\\Revit\" -f RevitNodes.dll -o \"C:\\Users\\SylvesterKnudsen\\Desktop\\TestMdOutputRevitNodes - Compressed\" -d \"C:\\Users\\SylvesterKnudsen\\Documents\\GitHub\\DynamoDictionary\\public\\data\\Dynamo_Nodes_Revit.json\" -l \"C:\\Users\\SylvesterKnudsen\\Desktop\\Logs\" -x \"C:\\Users\\SylvesterKnudsen\\Documents\\GitHub\\DynamoRevit\\src\\DynamoRevit\\Resources\\LayoutSpecs.json\" -w -s"
```

Comment:
file should probs not be committed to repo ?
or have a non user-specific path if that's possible ?